Lib9c/Currencies.cs

```diff
@@ -0,0 +1,70 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Libplanet;
+using Libplanet.Assets;
+using Nekoyume.TableData;
+
+namespace Nekoyume
+{
+    public static class Currencies
+    {
+        public static readonly Currency NCG = Currency.Legacy(
+            "NCG",
+            2,
+            new Address("0x47D082a115c63E7b58B1532d20E631538eaFADde"));
```

Comment:
It should be configurable because when the action logic uses `Currencies.NCG` it will not work in another forked network (because the minter will be different)

Reply:
@moreal Yes, you are right. And some other forked network can hard-fork the NCG's minters in this line.🙂
After the pluggable-lib9c is applied, use `Currencies.NCG` in another context. There are no uses of `Currencies.NCG` now.